src/main/java/de/hysky/skyblocker/skyblock/profileviewer/ProfileViewerScreen.java

```diff
@@ -0,0 +1,256 @@
+        profileViewerNavButtons.forEach(this::addDrawableChild);
+        for (ProfileViewerPage profileViewerPage : profileViewerPages) {
+            if (profileViewerPage != null && profileViewerPage.getButtons() != null) {
+                for (ClickableWidget button : profileViewerPage.getButtons()) {
+                    if (button != null) addDrawableChild(button);
+                }
+            }
+        }
+    }
+
+    public static void initClass() {
+        fetchCollectionsData(); // caching on launch
+
+        ClientCommandRegistrationCallback.EVENT.register((dispatcher, registryAccess) -> {
+            dispatcher.register(
+                    ClientCommandManager.literal(SkyblockerMod.NAMESPACE)
+                            .then(ClientCommandManager.literal("pv")
+                                    .then(ClientCommandManager.argument("username", StringArgumentType.string())
+                                            .executes(context -> {
+                                                String username = StringArgumentType.getString(context, "username");
+                                                Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                                return cmd.run(context);
+                                            })
+                                    )
+                                    .executes(context -> {
+                                        String username = MinecraftClient.getInstance().getSession().getUsername();
+                                        Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                        return cmd.run(context);
+                                    })
+                            )
+            );
+
+            dispatcher.register(
+                    ClientCommandManager.literal("pv")
+                            .then(ClientCommandManager.argument("username", StringArgumentType.string())
+                                    .executes(context -> {
+                                        String username = StringArgumentType.getString(context, "username");
+                                        Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                        return cmd.run(context);
+                                    })
+                            )
+                            .executes(context -> {
+                                String username = MinecraftClient.getInstance().getSession().getUsername();
+                                Command<FabricClientCommandSource> cmd = Scheduler.queueOpenScreenCommand(() -> new ProfileViewerScreen(username));
+                                return cmd.run(context);
+                            })
+            );
+        });
```

Comment:
`ArgumentBuilder#redirect` exists btw. `ExperienceCommand` has a good example. Also the formatting is a bit messed up.